docs/en/08_troubleshooting.md

```diff
@@ -95,3 +95,148 @@ _-: php: command not found
+    // Fourth retry attempt - Retry after 80 minutes
+    private static int $max_retry_attempts = 4;
+    private static int $initial_retry_delay = 600;
+    private static float $retry_falloff_multiplier = 2;
+    private static float $retry_falloff_multiplier_variance = 0;
+
+    // Retry pattern with spread
+    // First retry attempt - Retry after 8 to 12 minutes
+    // Second retry attempt - Retry after 6.4 to 14.4 minutes
+    // Third retry attempt - Retry after 5.1 to 27.4 minutes
+    // Fourth retry attempt - Retry after 4 to 38.4 minutes
+    private static int $max_retry_attempts = 4;
+    private static int $initial_retry_delay = 600;
+    private static float $retry_falloff_multiplier = 1;
+    private static float $retry_falloff_multiplier_variance = 0.2;
+}
+```
+
+#### Cluster breaking configuration
+
+This configuration is recommended for dealing with clusters of stuck jobs.
+A fixed retry delay typically doesn't help as all jobs will likely be retried at roughly the same time which will repeat the situation that caused the initial cluster to form - and therefore increase the chance of the job getting stuck again.
+
+This scenario is best handled by introducing random delay which spreads the jobs and thus eliminates the cluster.
+You might want to refine this configuration over multiple iterations - i.e. if multiple jobs are still getting stuck and failing around the same time, you might want to increase the retry_falloff_multiplier_variance to break up the cluster.
+Higher priority jobs should have lower offset and spread compared to lower priority jobs to minimise waiting times to process high priority jobs.
+
+```php
+namespace App\Jobs;
+
+use Symbiote\QueuedJobs\Services\AbstractQueuedJob;
+
+class MyJob extends AbstractQueuedJob
+{
+    // First retry attempt - Retry after 10 to 19.6 minutes
+    // Second retry attempt - Retry after 10 to 27.4 minutes
+    // Third retry attempt - Retry after 10 to 38.4 minutes
+    // Fourth retry attempt - Retry after 10 to 53.7 minutes
+    // Fifth retry attempt - Retry after 10 to 75.2 minutes
+    private static int $max_retry_attempts = 5;
+    private static int $initial_retry_delay = 600;
+    private static float $retry_falloff_multiplier = 1.2;
+    private static float $retry_falloff_multiplier_variance = 0.2;
+}
+```
+
+#### Global settings
+
+Global configuration is available on the `QueuedJobService` class:
+
+- `job_retry_buffer` represents separation of job retries and job processing mechanism to avoid potential edge cases, defaults to 1 minute
+- `job_retry_limit` how many stuck jobs can be retried per a single execution of `runQueue()`, set to `10` to disable job retries, defaults to `0` (globally disabled), recommended safe value is `10`
+- `job_retry_status_map` defines the job status transformation map, this allows to customise how the job statuses change during a job retry, defaults to `New` for `Broken` jobs and `Waiting` for `Paused` jobs
```

Comment:
Please include an example of setting `job_retry_status_map` (even just the default values will be fine) as it's not 100% clear from reading this documentation.